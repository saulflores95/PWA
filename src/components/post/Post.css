.title {
  font-size:3rem;
  color: gray;
}

.content {

}
