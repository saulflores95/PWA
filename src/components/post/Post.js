import React from 'react'
import styles from './Post.css'

const Post = () => (
  <div>
    <h2 className={styles.title}><a className={styles.link} href='#'>Blog Title</a></h2>
    <p className={styles.content}>Content Content Content Content Content Content Content Content Content Content Content Content Content </p>
  </div>
)

export default Post
