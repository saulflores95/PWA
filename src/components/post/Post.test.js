import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Post from './Post'

test('Post component should render the title not as a link', () => {
  const component = shallow(<Post title='My post' />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})

test('Post component should render the title as a link when the titleLink prop is true', () => {
  const component = shallow(<Post titleLink />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
