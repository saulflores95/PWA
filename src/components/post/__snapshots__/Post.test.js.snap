exports[`test Post component should render as expected 1`] = `
<div>
  <h2
    className="title">
    My Post
  </h2>
  <p
    className="content" />
</div>
`;

exports[`test Post component should render as expected 2`] = `
<div>
  <h2
    className="title">
    <Link
      activeClassName=""
      activeOnlyWhenExact={false}
      activeStyle={Object {}}
      className="link"
      isActive={[Function]}
      replace={false}
      style={Object {}}
      to="/post/undefined">
      My Post
    </Link>
  </h2>
  <p
    className="content" />
</div>
`;
